project/utils/constructVerificationEmailContent.php

```diff
@@ -21,13 +20,13 @@ function constructActivationEmailBody__FAKE($email, $userID) {
  * @param $userID
  * @return null|string
  */
-function constructActivationEmailBody($email, $userID) {
+function constructActivationEmailBody($email, $userID, $password) {
```

Comment:
This function is also used for sending email to single user. Do we want to send him email with his password too?